swift-sdk.xcodeproj/project.pbxproj

```diff
@@ -867,6 +876,7 @@
 				AC98294A20D9D65E00796DAA /* Foundation.framework */,
 			);
 			name = Frameworks;
+			path = ..;
```

Comment:
Why is this needed?

Reply:
This was to put `Frameworks` and `Products` under the `misc` folder, just for slightly better organization.